clipmenu

```diff
@@ -69,3 +71,5 @@ done
 if (( CM_OUTPUT_CLIP )); then
     cat "$file"
 fi
+
+exit $launcher_exit
```

Comment:
It doesn't matter, but to avoid the wrath of the linter, quote this please :-)

Reply:
ShellCheck was fine with it like this, but I'll quote just in case. :smile:

But if the variable is empty, quoting will produce an error in this case (see my comment on the other change).

Reply:
I added quoting around the variable, and a "not-empty" check before calling exit, so the quoting won't cause an errror.